tests/test_slam.py

```diff
@@ -46,6 +48,12 @@ async def test_get_latest_map_info(self):
         time = await self.slam.get_latest_map_info()
         assert time == MockSLAM.LAST_UPDATE
 
+    @pytest.mark.asyncio
+    async def test_get_properties(self):
+        (cloud_slam, mapping_mode) = await self.slam.get_properties()
+        assert cloud_slam == MockSLAM.CLOUD_SLAM
+        assert mapping_mode == MockSLAM.MAPPING_MODE
```

Comment:
Does `MockSLAM` have a `CLOUD_SLAM` and `MAPPING_MODE` object? I don't see it in `tests/mocks/services.py > MockSLAM`.

Reply:
Forgot to push that file, update should include it